panel/src/components/Layout/Items.vue

```diff
@@ -15,10 +15,10 @@
         v-for="(item, itemIndex) in items"
         :key="item.id || itemIndex"
         v-bind="item"
-        :class="{'k-draggable-item': item.sortable}"
+        :class="{'k-draggable-item': sortable && (item.sortable !== undefined ? item.sortable : true)}"
```

Comment:
```suggestion
        :class="{'k-draggable-item': sortable && (item.sortable || true)}"
```

Would this work as well?

Reply:
I thought so too at first but `false || true` returns `true`. We need `item.sortable ?? true` but our panel still doesn't support:
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Ahh of course, you are right!

Reply:
Why `Nullish coalescing operator` doesn't working. I can test on JSFiddle but in our panel throwing error on `npm run build`. Do you have any idea?

Reply:
We should fix this in way that item.sortable is already a boolean and the check is not needed.

Reply:
@bastianallgeier `item.sortable` is `boolean` on sections but `undefiened` on pages/files/users field.

Reply:
Then let's fix it on the PHP side

Reply:
With setting `item.sortable = true` as default on PHP for picker fields?

Reply:
@bastianallgeier I've push commit https://github.com/getkirby/kirby/pull/3557/commits/ecf74c11d609e06837d26978a6eac3302e5efca6. Like that?